folding-schemes/src/frontend/noir/test_folder/test_circuit.json

```diff
@@ -0,0 +1 @@
+{"noir_version":"0.30.0+af57471035e4fa7eaffa71693219df6d029dbcde","hash":3284229809271179646,"abi":{"parameters":[{"name":"public_inputs","type":{"kind":"array","length":2,"type":{"kind":"field"}},"visibility":"public"},{"name":"private_inputs","type":{"kind":"array","length":2,"type":{"kind":"field"}},"visibility":"private"}],"param_witnesses":{"private_inputs":[{"start":2,"end":4}],"public_inputs":[{"start":0,"end":2}]},"return_type":{"abi_type":{"kind":"array","length":2,"type":{"kind":"field"}},"visibility":"public"},"return_witnesses":[6,7],"error_types":{}},"bytecode":"H4sIAAAAAAAA/9WSUQ7DIAiG1dl5HRCt8LarzMze/wibmcuapm+lD/0SQnj5w0ew5kv4lDd/7OiP0eEY2DPcTi7BnFIrsSHhE6JUzpBynRkZM+dXZKLGiYtUKSCYqOGShZYR7PV2hDO8e97tBO9J2fuHU/b3is73izhPis5B0bn/oN84u/GfbnUPu5r7zYPZ4Q0o50beOQQAAA==","debug_symbols":"TclRCoAgDADQu+y7E3SViJimMZAtSoMYu7uCfvj5eApncOU6iKO8sG4KSTxmEm5SW+DDh9ClMBgL+2nzf/ex3So=","file_map":{},"names":["main"]}
```

Comment:
One question from my ignorance about Noir, are these kind of files necessary? or can they be generated locally avoiding to upload them to github?

Reply:
Good question, the user can generate those locally. I added a `compile.sh` script, modified the CI and the tests to actually reflect that. [ca34f2c](https://github.com/privacy-scaling-explorations/sonobe/pull/131/commits/ca34f2ca716c5c62ad99c23bcb831d4c8a83a76c)

Reply:
oh nice! Thanks for updating it! ^^